static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -454,6 +454,8 @@ define([
             if (mobileAdSizes && mobileAdSizes.indexOf(adSizes.stickyMpu) > -1) {
                 stickyMpu($adSlot);
             }
+        } else if ($adSlot.hasClass('ad-slot--facebook')) {
+            $adSlot.addClass('ad-slot--fluid');
```

Comment:
Good point, I had to verify but no. The promise resolves when the callback is called, which can only happen after the elements has been added to the DOM.